### ISSUE TITLE
[pt-br] user feedback 

### ISSUE THREAD

**Issue by tiff** [languagetool-org/languagetool#4564]
See https://twitter.com/monodraminha/status/1368626540779630596

**Comment by marcoagpinto** [comment]
Yes, the suggestions says: "If it is a PhD thesis, verify if the style of writing is the appropriated blah blah."

Maybe I should change the rule to picky?

**Comment by tiff** [comment]
Thanks for checking. Sounds good

**Comment by marcoagpinto** [comment]
@tiff 

In LibreOffice, where is the picky option in "LanguageTool → Options"?

I can't find it.

I want all picky rules to work for me in LibreOffice.

Thanks!

**Comment by tiff** [comment]
I don‘t know. I‘m not using/developing the LibreOffice plugin. Sorry
